tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -40,6 +40,46 @@ namespace ROOT {
 namespace Experimental {
 namespace Detail {
 
+std::string inline GetDaosError(int err)
```

Comment:
IIRC, `d_errstr()` returns a `const char *` to a buffer that we don't own; thus, can can safely return a `std::string_view` instead.